deploy/kubernetes/driver/kubernetes/overlays/stage/node-server-images.yaml

```diff
@@ -13,5 +13,3 @@ spec:
           image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.2.0
         - name: liveness-probe
           image: k8s.gcr.io/sig-storage/livenessprobe:v2.3.0
-        - name: csi-resizer
```

Comment:
Why are we removing resizer ?

Reply:
I checked with Meghna and she said this container is not  required in node pods